plugins/jira/commands/issues-by-component.md

```
@@ -0,0 +1,674 @@
+---
+description: List and analyze JIRA issues organized by component with flexible filtering
+argument-hint: <project-key> [time-period] [--component name] [--assignee username] [--reporter username] [--status status] [--search term] [--search-description]
+---
+
+## Name
+jira:issues-by-component
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)